introduction/HANDSON.md

```diff
@@ -226,3 +267,52 @@ spec:
   selector:
     app: nginx
 ```
+
+# Namespaces
+
+```yaml
+# Namespace manifest file nginx-namespace-dev.yaml
+# If using Vim type ":set paste" before pasting this
+{
+  "apiVersion": "v1",
+  "kind": "Namespace",
+  "metadata": {
+    "name": "development",
+    "labels": {
+      "name": "development"
+    }
+  }
+}
+```
+
+```yaml
```

Comment:
You are using JSON for the file format but specifying yaml as the markdown format, is that expected?